danger_code_ws/1.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <cstdio>
+
+int main() {
+    char buffer[10];
+
+    std::cout << "Enter a string: ";
+    fgets(buffer, sizeof(buffer), stdin); // Safer alternative to gets
```

Comment:
Replace `gets(buffer)` with `fgets(buffer, sizeof(buffer), stdin)` to avoid buffer overflow vulnerabilities.